integrationtests/test_i18n.py

```diff
@@ -2,38 +2,47 @@
 from integrationtests.browser import Browser
 
 
-_locale_names = {"en": "English", "el": "Ελληνικά"}
-
-
-def login(
-    browser: Browser,
-    email: str,
-    password: str,
-    locale_id: str = "en",
-    after_login_locale_id: str = "en",
-) -> None:
+def login(browser: Browser, email: str, password: str) -> None:
     """Log in through `/account/login` as the given user.
     The login page must be in `locale_id`, while the workflows page must be in `after_login_locale_id`
     """
     browser.visit("/account/login")
     browser.fill_in("login", email)
     browser.fill_in("password", password)
-    login_button_text = {"en": "Sign In", "el": "Σύνδεση"}
-    browser.click_button(login_button_text[locale_id])
-    my_workflows_text = {"en": "MY WORKFLOWS", "el": "ΟΙ ΡΟΈΣ ΕΡΓΑΣΙΏΝ ΜΟΥ"}
-    browser.wait_for_element(
-        "a", text=my_workflows_text[after_login_locale_id], wait=True
-    )
+    browser.click_whatever('.account_form.login button[type="submit"]')
+    browser.wait_for_element(".create-workflow")
 
 
-def logout(browser: Browser, locale_id: str = "en") -> None:
+def logout(browser: Browser) -> None:
     """Log out through `/account/logout` as the given user.
     
     The logout page must be in `locale_id`
     """
     browser.visit("/account/logout")
-    logout_button_text = {"en": "Log out", "el": "Αποσύνδεση"}
-    browser.click_button(logout_button_text[locale_id])
+    browser.click_whatever('.account_form button[type="submit"]')
+    browser.wait_for_element(".account_form.login")
+
+
+def switch_locale_django(browser: Browser, to_locale_name: str):
+    browser.click_whatever("a#locale-switcher-dropdown", wait=True)
+    browser.click_button(to_locale_name, wait=True)
+
+    # check that the language has indeed changed
+    browser.wait_for_element("a#locale-switcher-dropdown", text=to_locale_name)
+
+
+def switch_locale_react(browser: Browser, to_locale_name: str):
+    browser.click_whatever(".navbar .dropdown button", wait=True)
```

Comment:
Why is there a `wait=True` here? Do we ever call `switch_locale_react()` when the hamburger-menu button hasn't loaded?

With `wait=True`, the sentence is: "Wait for the hamburger-menu button to appear, then click it."

Without, the sentence is: "Click the hamburger-menu button."

I think `wait=True` should be nixed. Otherwise, we certainly need a comment explaining why we're waiting. (Our standard practice is to _always_ put a comment when using `wait=True`; but I feel the next two lines in this function don't need comments because it's perfectly clear what we're waiting for.)